sign/ed448/ed448.go

```diff
@@ -186,9 +191,10 @@ func Verify(public PublicKey, message, context, signature []byte) bool {
 	S := &goldilocks.Scalar{}
 	S.FromBytes(signature[paramB:])
 
-	encR := (&[paramB]byte{})[:]
 	P.Neg()
-	_ = goldilocks.Curve{}.CombinedMult(S, k, P).ToBytes(encR)
+	Q := &goldilocks.Point{}
+	goldilocks.Curve{}.CombinedMult(Q, S, k, P)
+	encR, _ := Q.MarshalBinary()
```

Comment:
Is there a reason why the error is being ignored here?

Reply:
It won't return an error, same as above.